File: apollo/src/main/java/io/muun/apollo/domain/errors/CountryNotSupportedError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class CountryNotSupportedError extends UserFacingError {

    public CountryNotSupportedError() {
        super(UserFacingErrorMessages.INSTANCE.countryNotSupported());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/lnurl/CountryNotSupportedError.kt
package io.muun.apollo.domain.errors.lnurl

import io.muun.apollo.domain.errors.ErrorClassification
import io.muun.apollo.domain.errors.MuunError

class CountryNotSupportedError(message: String, domain: String) : MuunError() {

    override val classification = ErrorClassification.EXPECTED

    init {
        metadata["service"] = domain
        metadata["message"] = message
    }
}
